src/web/gwc/src/test/java/org/geoserver/gwc/web/layer/CachedLayersPageTest.java

```diff
@@ -199,4 +199,25 @@ public void testAutoTileCachingTabSelection() {
         // UI should navigate to ResourceConfiguration Page with Tiles tab selected
         tester.assertComponent("publishedinfo:tabs:panel", LayerCacheOptionsTabPanel.class);
     }
+    
+    @Test
+    public void testGWCClean() {
+        // This asserts GWC integration from GUI
+        CachedLayersPage page = new CachedLayersPage();
+        
+        // load tiles layer list page
+        tester.startPage(page);
+        tester.assertRenderedPage(CachedLayersPage.class);
+
+        //        // click on the Empty All Link
+        tester.clickLink("headerPanel:clearGwcLink", true);
+
+        // assert the dialog shows up
+        tester.assertVisible("dialog");
+
+        // click submit
+        tester.clickLink("dialog:dialog:content:form:submit", true);
+
+        tester.assertNoErrorMessage();
```

Comment:
Check that all tiles are gone please (fill some in advance, there are other tests doing it)

Reply:
will do

Reply:
@aaime I am unable to find those tests. Apparently only truncate calls are integrated, and for the Seed and Reseed, the GS-GUI opens up GWC-GUI.

Reply:
See this test, forces creation of a tile to check if it gets deleted on style rename:
https://github.com/geoserver/geoserver/blob/1a86e81df6fb3e1dd96944fd983fd4bf90a6051e/src/gwc/src/test/java/org/geoserver/gwc/GWCIntegrationTest.java#L721

Reply:
This change is still outstanding right?

Reply:
that test has been added here : https://github.com/geoserver/geoserver/pull/3922/commits/3cbf3279e9204105b5a7457531737f3e3c301231#diff-b860552a5e2a8a1a22d887bc2fc9956c

this is GUI test only asserts UI functions

Reply:
@aaime here https://github.com/geoserver/geoserver/pull/3922/commits/19eb75db28edd30b8e8be599f88f631564972ef6#diff-b860552a5e2a8a1a22d887bc2fc9956cR2071